app/services/action_service.rb

```diff
@@ -47,8 +47,9 @@ def send_email_transcript(emails)
 
   private
 
-  def agent_belongs_to_account?(agent_ids)
-    @account.agents.pluck(:id).include?(agent_ids[0])
+  def agent_belongs_to_inbox?(agent_ids)
+    member_ids = @conversation.inbox.members.pluck(:user_id)
```

Comment:
This + admins of the account can be added to the conversation.. I thought that was clear, My mistake for not communicating the permissions properly. The intention was to check whether the agent was assignable or not.